src/main/java/org/primefaces/component/dataview/DataView.java

```diff
@@ -182,7 +183,7 @@ public void resetMultiViewState() {
     public void restoreMultiViewState() {
         DataViewState viewState = getMultiViewState(false);
         if (viewState != null) {
-            if (viewState.getLayout() != null && viewState.getLayout().length() > 0) {
+            if (!LangUtils.isValueBlank(viewState.getLayout())) {
```

Comment:
We have the more readable `LangUtils.isNotBlank` instead of `!LangUtils.isValueBlank`